src/cursor/cursor.ts

```diff
@@ -1307,10 +1307,11 @@ export class Cursor<
   explain(callback?: Callback): Promise<unknown> | void {
     // NOTE: the next line includes a special case for operations which do not
     //       subclass `CommandOperationV2`. To be removed asap.
-    if (this.operation && this.operation.cmd == null) {
-      this.operation.options.explain = true;
-      return executeOperation(this.topology, this.operation as any, callback);
-    }
+    // TODO: uncomment/fix this when cursor explain is re-implemented
+    // if (this.operation && this.operation.cmd == null) {
+    //   this.operation.options.explain = true;
+    //   return executeOperation(this.topology, this.operation as any, callback);
+    // }
```

Comment:
Do you know if there is a ticket for this?

Reply:
Yep, it's one I am working on concurrently (and will hopefully merge immediately after this). I can still put the ticket number for convenience, though.